### PR TITLE
Extend attibution

### PR DESCRIPTION
**Pull request by hartwork**

### CODE REVIEW

File: ttyplot.1 [hunk 1]
```diff
@@ -283,8 +283,11 @@ Prometheus node exporter disk throughput for
 .Ed
 .Sh AUTHORS
 .Nm
-as written by
-.An Antoni Sawicki Aq Mt tenox@google.com .
+is written by ..
+.br
+.An Antoni Sawicki Aq Mt tenox@google.com
+.An Edgar Bonet Aq Mt linux@edgar-bonet.org
+.An Sebastian Pipping Aq Mt sebastian@pipping.org
```
[the remark this answers is not in the document]
Comment (hartwork) [reply ttyplot.1:290]: @edgar-bonet I compared the before and after…

#### Before (in this pull request)
before_Screenshot_20231127_215741

#### After (your suggestion + trailing full stop)

after_Screenshot_20231127_215656

…and while I find that "Before" doesn't win prices in beauty, the fact that we have no newline before Antoni's name "discriminates" against him a bit in the sense that he is now a lot less visible than us and he "needs more work to find".

How would you feel about a compromise with "," (comma), "." (full stop), and "and" but with a newline before Antoni's name also? With or without ".." as you prefer. What do you think? Other ideas? (I tried a single long line version and I decided against that approach early for myself, I should maybe add.)

[the remark this answers is not in the document]
Comment (edgar-bonet) [reply ttyplot.1:290]: I see your point. I am fine with breaking the line before Antoni, but the two dots look more like a formatting error than like regular punctuation. A colon would make more sense here:

**ttyplot** is written by:  
Antoni Sawicki <tenox@google.com>,  
Edgar Bonet <linux@edgar-bonet.org> and  
Sebastian Pipping <sebastian@pipping.org>.

[the remark this answers is not in the document]
Comment (hartwork) [reply ttyplot.1:290]: @edgar-bonet I'm happy to turn it into that, give me a few seconds…

[the remark this answers is not in the document]
Comment (hartwork) [reply ttyplot.1]: @edgar-bonet pushed, this is what I see:

latest

What do you think?

[the remark this answers is not in the document]
Comment (edgar-bonet) [reply ttyplot.1]: I like it!

[the remark this answers is not in the document]
Comment (hartwork) [reply ttyplot.1]: @edgar-bonet cool!

**Pull request merged**